src/dat/gui/GUI.js

```diff
@@ -557,6 +561,27 @@ common.extend(
     },
 
     /**
+     * Adds a new custom controller to the GUI.
+     *
+     * @param object
+     * @param property
+     * @returns {Controller} The controller that was added to the GUI.
+     * @instance
+     *
+     */
+    addCustomController: function(object, property) {
+    	return add(
+		  this,
+		  object,
+		  property,
+		  {
+		  	custom: true,
```

Comment:
Do we need the `custom: true` parameter, if we can already check that the object is an instance of CustomController?

Reply:
Yes. I have removed it